cms/models.py

```diff
@@ -40,17 +72,77 @@ class HomePage(Page):
         help_text="The subtitle text to display in the hero section of the home page.",
     )
 
+    product_section_title = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text="The tile text to display in the product cards section of the home page.",
+    )
+
     content_panels = Page.content_panels + [
         ImageChooserPanel("hero"),
         FieldPanel("hero_title"),
         FieldPanel("hero_subtitle"),
+        FieldPanel("product_section_title"),
+        InlinePanel("featured_products", label="Featured Products"),
     ]
+
     parent_page_types = [Page]
     subpage_types = [
-        "CoursePage",
         "ResourcePage",
     ]
 
+    def _get_child_page_of_type(self, cls):
+        """Gets the first child page of the given type if it exists"""
+        child = self.get_children().type(cls).live().first()
+        return child.specific if child else None
+
+    @property
+    def product_cards(self):
+        pages = []
+        for page in self.featured_products.all():
+            if page.course_product_page:
+                pages.append(page.course_product_page.specific)
+        return pages
+
+    def get_context(self, request, *args, **kwargs):
+        return {
+            **super().get_context(request),
+            "product_cards_section_title": self.product_section_title,
+            "product_cards": self.product_cards,
+        }
+
+
+class HomeProductLink(Orderable):
+    """
+    Home and ProductPage Link
+    """
+
+    page = ParentalKey(
+        HomePage, on_delete=models.CASCADE, related_name="featured_products"
+    )
+
+    course_product_page = models.ForeignKey(
+        "wagtailcore.Page",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    panels = [
+        PageChooserPanel("course_product_page", "cms.CoursePage"),
+    ]
+
+
+class CourseIndexPage(CourseObjectIndexPage):
```

Comment:
We're probably going to introduce this change in a future PR, but is there any reason it's necessary here? When we do introduce it, we're probably going to add this index page if it doesn't exist, and introduce a data migration to move all existing course pages to be children of it, just like we did in xpro. If this isn't necessary for this PR, please remove it

Reply:
I can remove it but at the moment it seems me mess when I saw a lot of `CoursePage` under `HomePage`. Ok make sense to do it separate PR but it should be.